projects/Mallard/src/components/article/types/article.tsx

```diff
@@ -195,18 +192,15 @@ const Article = ({
 	const previewParam = isPreview(apiUrl) ? `?frontId=${front}` : '';
 
 	// if webUrl is undefined then we attempt to fetch a url to use for sharing
-	const [shareUrl, setShareUrl] = useState(article.webUrl);
+	const [shareUrl, setShareUrl] = useState(article.webUrl ?? '');
 	// we can only attempt to fetch the url if connected
 	const client = useApolloClient();
 	const data = client.readQuery<{ netInfo: { isConnected: boolean } }>({
 		query: gql('{ netInfo @client { isConnected @client } }'),
 	});
 	const isConnected = data?.netInfo?.isConnected;
-	const shareUrlFetchEnabled =
-		!article.webUrl &&
-		isConnected &&
-		// TODO: remove remote switch once we are happy this feature is stable
-		remoteConfigService.getBoolean('generate_share_url');
+
+	const shareUrlFetchEnabled = isConnected;
```

Comment:
Do we need isConnected or could this just be `const shareUrlFetchEnabled = data?.netinfo?.isConnected;` ?

Reply:
lol yeah 🤯

Reply:
or just `isConnected`